cns/api.go

```diff
@@ -56,6 +57,11 @@ type IPConfigurationStatus struct {
 	OrchestratorContext json.RawMessage
 }
 
+func (i IPConfigurationStatus) ToString() string {
```

Comment:
If the method is renamed to just `String`, it will meet the `Stringer` interface which is convenient in multiple places, including formatting. For example, 

```
fmt.Sprintf("%v", ipConfig)
```
would work as expected since Sprintf understands the `String()` method

Reply:
Makes sense, I used the incorrect name (well it is correct in C# ;))

---
In reply to: [585062394](https://github.com/Azure/azure-container-networking/pull/806#discussion_r585062394) [](ancestors = 585062394)